sdk/Sdk.Generators/DiagnosticDescriptors.cs

```diff
@@ -45,7 +45,7 @@ private static DiagnosticDescriptor Create(string id, string title, string messa
         public static DiagnosticDescriptor MultipleHttpResponseTypes { get; }
                   = Create(id: "AZFW0007",
                     title: "Symbol could not be found in user compilation.",
-                    messageFormat: "Found multiple public properties with type '{0}' defined in output type '{1}'.Only one HTTP response binding type is supported in your return type definition.",
+                    messageFormat: "Found multiple public properties of type HttpResponseData defined in return type '{0}'.Only one HTTP response binding type is supported in your return type definition.",
```

Comment:
Original error message was taken from [legacy generator](https://github.com/Azure/azure-functions-dotnet-worker/blob/b152392e347acbfd97a55fae7f887e55ed5f3542/sdk/Sdk/FunctionMetadataGenerator.cs#L316). I think it's _unlikely_ for this diagnostic error to be hit, but going for parity with legacy generator on error handling.